tests/ui_tests/test_jobs_page.py

```diff
@@ -185,3 +318,45 @@ def _check_job_info_in_popup(page: JobListPage, expected_info: dict):
     """Get job info from popup and check it"""
     with page.header.open_jobs_popup():
         wait_and_assert_ui_info({**expected_info}, page.get_task_info_from_popup)
+
+
+@allure.step('Run 6 success and 5 failed actions on 11 hosts')
+def _run_actions_on_hosts(hosts: List[Host]):
```

Comment:
I think will be better to remove hardcoded numbers and use values from function args

Reply:
Sure